spec/support/schemas/claims_api/v2/forms/disability/submission.json

```diff
@@ -517,10 +517,10 @@
                     "example": "249470"
                   },
                   "approximateDate": {
-                      "description": "Approximate date disability began. \n Format can be either MM-DD-YYYY or MM-YYYY",
-                      "type": "string",
-                      "pattern": "^(0[1-9]|1[0-2])(-(0[1-9]|[1-2][0-9]|3[0-1]))?-\\d{4}$|^(0[1-9]|1[0-2])-\\d{4}$",
-                      "example": "03-12-2019 or 03-2019"
+                    "description": "Approximate date disability began. \n Format can be either MM-DD-YYYY or MM-YYYY or YYYY",
+                    "type": "string",
+                    "pattern": "^(0[1-9]|1[0-2])(-(0[1-9]|[1-2][0-9]|3[0-1]))?-\\d{4}$|^(0[1-9]|1[0-2])-\\d{4}$|^\\d{4}$",
+                    "example": "03-12-2019 or 03-2019 or 2019"
```

Comment:
these changes are highlighted incorrectly, I believe because the values are the same when moved over so it misunderstood where the addition was.